pkg/common/constants.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+const (
+	// Keys for Storage Class Parameters
+	ParameterKeyZone = "zone"
+	ParameterKeyType = "type"
+
+	// Keys for Topology
+	TopologyKeyZone = "com.google.gcp-compute-persistent-disk-csi-driver.topology/zone"
```

Comment:
I think we should not put the driver name in the key.  Filestore driver should be able to share this key too.

Reply:
I thought we didn't want collisions between topology keys? I may be misunderstanding what this is used for exactly

Reply:
We want to be able to share topology keys amongst drivers from the same vendor.
1. To avoid proliferation of labels on Node objects
2. To allow users to use a common set of topology labels in their Pods.